Amplify.xcodeproj/project.pbxproj

```diff
@@ -2925,7 +2916,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\"${PODS_ROOT}/SwiftFormat/CommandLineTool/swiftformat\" \"${SRCROOT}/AmplifyTestCommon\"\n";
+			shellScript = "\"${PODS_ROOT}/SwiftFormat/CommandLineTool/swiftformat\" --swiftversion \"$SWIFT_VERSION\" \"${SRCROOT}/AmplifyTestCommon\"\n";
```

Comment:
Do you mean "Why are we doing SwiftFormat" or "Why are we passing `--swift-version`? If the latter, it's because SwiftFormat needs to know what syntax flavors it's working with to be able to parse & reformat the code appropriately.